coffee-model/coffee-model-base/src/main/java/hu/icellmobilsoft/coffee/model/base/IIdentifiedAuditEntity.java

```diff
@@ -0,0 +1,114 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.coffee.model.base;
+
+/**
+ * Interface for the identified audit entities. Contains setters, getters for id, version, and for the basic audit fields.
+ *
+ * @param <DATE> The type of the date. E.g.: {@link java.util.Date}, {@link java.time.OffsetDateTime}, {@link java.time.Instant}
+ * @author martin.nagy
+ * @since 1.8.0
+ */
+public interface IIdentifiedAuditEntity<DATE> {
+
+    /**
+     * Returns the id of the entity
+     *
+     * @return the id of the entity
+     */
+    String getId();
```

Comment:
Ez genericum a AbstractIdentifiedAuditEntity osztalyokban igy most akkor kicsit ellentmondasos

Reply:
Legyen az ID-ra is generikus paraméter? Vagy a user field-ekre gondolsz?

Reply:
Megoldas sok lehet talan az id-re is a generikum a leggyorsabb, vagy kiszervezni max ujabb interfeszbe de az most kicsit feleslegesnek tunik mar.